app/models/category.rb

```diff
@@ -46,10 +46,22 @@ class Category < ActiveRecord::Base
   has_many :topic_timers, dependent: :destroy
   has_many :upload_references, as: :target, dependent: :destroy
 
-  has_one :category_setting, dependent: :destroy
+  # TODO: This should really be `category_setting`,but that conflicts with
+  #       an association added in `discourse-topic-voting` plugin. We should
+  #       have the plugin extend this core model and then rename this.
+  has_one :category_core_setting, class_name: "CategorySetting", dependent: :destroy
```

Comment:
I normally wouldn't endorse this kind of "bag-o'-words" naming, but I don't want to lose the connection to `category_setting` in the plugin.

Reply:
I've fixed the plugin so you shouldn't have to do this anymore. 

See https://github.com/discourse/discourse-topic-voting/pull/148

Reply:
Better! 🙂